code/modules/mob/living/carbon/carbon.dm

```diff
@@ -340,14 +340,13 @@
 
 	sight = initial(sight)
 	lighting_alpha = initial(lighting_alpha)
-	see_in_dark = species.darksight
```

Comment:
just set it to initial or something. While presumably we should always have a species, if we somehow dont we should probably reset it.